client/src/templates/Challenges/classic/editor.tsx

```diff
@@ -705,7 +646,20 @@ const Editor = (props: EditorProps): JSX.Element => {
     }
   };
 
-  function decorateForbiddenRanges(editableRegion: number[]) {
+  function initializeProjectFeatures() {
+    const editor = data.editor;
+    if (editor) {
+      initializeRegions(getEditableRegionFromRedux());
+      addWidgetsToRegions(editor);
+    }
+  }
+
+  function isProject() {
+    const editableRegionBoundaries = getEditableRegionFromRedux();
+    return editableRegionBoundaries.length === 2;
+  }
```

Comment:
I do not understand this function. What about the certification projects?

Reply:
In theory we'll need the editable regions to show the "jaws", and would position them at the top and bottom of the HTML maybe

Reply:
Hmm, good point.  We don't have a flag for the new challenges vs normal ones, but we do know that projects steps have a single editable region, so this checks to see if it exists and that there is only one.  The reason for the abstraction is in case we change what we use determine if it's a project step

How about `isProjectBasedStep` for the function name?